src/coreclr/vm/prestub.cpp

```diff
@@ -1991,7 +1991,7 @@ extern "C" PCODE STDCALL PreStubWorker(TransitionBlock* pTransitionBlock, Method
 }
 
 #ifdef FEATURE_INTERPRETER
-extern "C" void* STDCALL ExecuteInterpretedMethod(TransitionBlock* pTransitionBlock, TADDR byteCodeAddr, void* retBuff)
+extern "C" void* STDCALL ExecuteInterpretedMethodWithArgs(TransitionBlock* pTransitionBlock, TADDR byteCodeAddr, int8_t* pArgs, size_t size, void* retBuff)
```

Comment:
I would prefer keeping the ExecuteInterpretedMethod signature untouched and let this ExecuteInterpretedMethodWithArgs just get the threadContext->pStackPointer, copy the args there and then call the ExecuteInterpretedMethod. That way, it would behave in a way close to how the jit to interpreter calls work for other platforms.